### PR TITLE
Fix typos and add color to back link

### PR DESCRIPTION
**Pull request by gruberb**
I forgot to merge the latest changes in this PR (https://github.com/TheTorProject/ooni-web/pull/87)

- Color the back link blue
- Ordering of the tests in the "Tor" section
- Underline links while hovering

### CODE REVIEW

**Comment by gruberb** [comment]
Maybe @agrabeli can have a look at this and merge it

File: content/nettest/vanilla-tor.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: "Vanilla Tor"
 short_description: "This test examines the reachability of the Tor network (which is designed for online anonymity
 and censorship circumvention)."
 groups: ["tor"]
+date: "0000-01-01"
```
Comment (darkk) [inline content/nettest/vanilla-tor.md]: Do I understand it correctly, that this fake date is used for sorting?

Comment (gruberb) [inline content/nettest/vanilla-tor.md]: Yes! It was a straight-forward solution to order the posts. There is another options with "weight" and implementing taxonomies in the config.toml. It's of course up for discussion to implement it differently.

**Comment by hellais** [comment]
In order to merge this it will have to be rebased onto master.

**Comment by hellais** [comment]
Thanks!

They have been merged. Closing!

**Pull request closed without merging**